tasks/dnsmasq.yml

```diff
@@ -49,6 +49,11 @@
   when: ansible_service_mgr == "systemd"
   block:
 
+    - name: Check if systemd-resolved service exists
+      stat:
+        path: /lib/systemd/system/systemd-resolved.service
+      register: systemd_resolved_service
```

Comment:
Note: we could also potentially change this to use `service_facts`, but let's consider that in a follow-up.

Reply:
I think it should be a separate PR